phpcs.xml.dist

```diff
@@ -58,6 +58,7 @@
 	<!-- The main plugin file should be parsable by PHP < 5.6. -->
 	<rule ref="Generic.Arrays.DisallowLongArraySyntax.Found">
 		<exclude-pattern>web-stories.php</exclude-pattern>
+		<exclude-pattern>Web_Stories_Compatibility.php</exclude-pattern>
```

Comment:
This class should not live in the root folder. let's move it to some `compat` folder or something.

Note: any new folder or file in the root should be added to `.distignore`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5212/commits/79feede30971484f1f72f7c2489605dd29401eed